recipe/meta.yaml

```diff
@@ -11,11 +11,12 @@ package:
 
 source:
   url: https://github.com/dmlc/xgboost/releases/download/v{{ version }}/xgboost-{{ version }}.tar.gz
-  sha256: a84fc7d9846c24659a2ad16788a7eefa9640b19eea9bbc65f30e0a9d53c52453
+  sha256: 90b1b7b770803299b337dd9b9206760d9c16f418403c77acce74b350c6427667
   patches:
     # xgboost patches
-    - patches/0001-Enable-latest-libcxx-on-MacOS.patch  # [osx]
-    - patches/0002-Mark-wheels-as-any-platform-compatible.patch
+    - patches/0001-Enable-latest-libcxx-on-MacOS.patch                # [osx]
+    - patches/0002-Remove-nvidia-nccl-cu12-from-pyproject.toml.patch
```

Comment:
It looks like the NCCL wheel dependency removal patch is needed again. Even though it wasn't needed in the 2.1.2 release

https://github.com/conda-forge/xgboost-feedstock/pull/195#discussion_r1816185240

Reply:
@hcho3 @trivialfis , what is the generation process for artifacts attached to the GitHub release (like [2.1.3]( https://github.com/dmlc/xgboost/releases/tag/v2.1.3 ))?

Reply:
It looks like we generate two different files with the same name `xgboost-2.1.3.tar.gz` from [this script](https://github.com/dmlc/xgboost/blob/master/dev/release-artifacts.py). This is super confusing and we should remove one of them.

Reply:
Filed as upstream issue: https://github.com/dmlc/xgboost/issues/11045